helm/azuredisk-csi-driver-app/templates/azuredisk-controller/pss-exceptions.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v2alpha1
```

Comment:
For reference i think we should not merge to main any `Kyverno PolicyException` for any APP we install on WCs since we don't have the kyverno CRD on those clusters and it will make the chart installation fail

for reference: https://gigantic.slack.com/archives/C02HLSDH3DZ/p1686577261547249